runelite-client/src/main/java/net/runelite/client/plugins/kourendlibrary/KourendLibraryPanel.java

```diff
@@ -86,16 +86,16 @@ void init()
 		c.gridx = 0;
 		c.gridy = 0;
 		Stream.of(Book.values())
-			.filter(b -> !b.isDarkManuscript())
-			.filter(b -> !config.hideVarlamoreEnvoy() || b != Book.VARLAMORE_ENVOY)
-			.sorted(Comparator.comparing(Book::getShortName))
-			.forEach(b ->
-			{
-				BookPanel p = new BookPanel(b);
-				bookPanels.put(b, p);
-				books.add(p, c);
-				c.gridy++;
-			});
+				.filter(b -> !b.isDarkManuscript())
+				.filter(b -> !config.hideVarlamoreEnvoy() || b != Book.VARLAMORE_ENVOY)
+				.sorted(Comparator.comparing(Book::getShortName))
+				.forEach(b ->
+				{
+					BookPanel p = new BookPanel(b);
+					bookPanels.put(b, p);
+					books.add(p, c);
+					c.gridy++;
+				});
```

Comment:
Revert the indenting that happened here.
```suggestion
			.filter(b -> !b.isDarkManuscript())
			.filter(b -> !config.hideVarlamoreEnvoy() || b != Book.VARLAMORE_ENVOY)
			.sorted(Comparator.comparing(Book::getShortName))
			.forEach(b ->
			{
				BookPanel p = new BookPanel(b);
				bookPanels.put(b, p);
				books.add(p, c);
				c.gridy++;
			});
```

Reply:
Also, I believe we discussed that the panel should hide the Varlamore Envoy under the same circumstances it wouldn't be visible in the overlay?

Reply:
My most recent commit now updates the panel as well as the overlay.